data/jobs.txt

```diff
@@ -1477,6 +1477,76 @@ mission "Core Pirate Attack"
 	on complete
 		payment 200000
 		dialog "The government of <planet> pays you <payment> for helping to drive off the pirates."
+		
+mission "Raider Attack 1"
+	name "Defend <planet>"
+	description "Defeat an alien raid on <destination>."
+	minor
+	repeat
+	to offer
+		"combat rating" > 550
+		random < 7
+		has "main plot: completed"
+	source
+		planet "Sundive"
+		planet "Shangri-La"
+		planet "Nimbus"
+	npc evade
+		government "Korath"
+		personality timid unconstrained uninterested
```

Comment:
As with the above Core Pirate Attack missions, your opponent NPCs should also have `target harvets plunders`